tmpl/package_notwindows.tmpl

```diff
@@ -0,0 +1,106 @@
+//glow:keepspace
+// +build !windows
+
+// Code generated by glow (https://github.com/go-gl/glow). DO NOT EDIT.
+
+package {{.Name}}
+//glow:rmspace
+
+{{define "paramsCDecl"}}{{range $i, $p := .}}{{if ne $i 0}}, {{end}}{{$p.Type.CType}} {{$p.CName}}{{end}}{{end}}
+{{define "paramsCCall"}}{{range $i, $p := .}}{{if ne $i 0}}, {{end}}{{if $p.Type.IsDebugProc}}glowCDebugCallback{{else}}{{$p.CName}}{{end}}{{end}}{{end}}
+
+{{define "paramsGoDecl"}}{{range $i, $p := .}}{{if ne $i 0}}, {{end}}{{$p.GoName}} {{$p.Type.GoType}}{{end}}{{end}}
+{{define "paramsGoCall"}}{{range $i, $p := .}}{{if ne $i 0}}, {{end}}{{$p.Type.ConvertGoToC $p.GoName}}{{end}}{{end}}
+
+// #cgo darwin        LDFLAGS: -framework OpenGL
+// #cgo linux freebsd LDFLAGS: -lGL
+// #cgo windows       LDFLAGS: -lopengl32
+//
+// #if defined(_WIN32) && !defined(APIENTRY) && !defined(__CYGWIN__) && !defined(__SCITECH_SNAP__)
```

Comment:
presumably these windows-specific bits are no longer necessary in the not-windows files?

Reply:
True.

Reply:
Done.